server/src/main/kotlin/org/itmodreamteam/myrest/server/service/restaurant/RestaurantServiceImpl.kt

```diff
@@ -26,16 +26,15 @@ class RestaurantServiceImpl(
     override fun register(newRestaurant: RestaurantRegistrationInfo, user: User): RestaurantInfo {
         val existingRestaurant = restaurantRepository.findByName(newRestaurant.name)
         val admins = userRepository.findByRole(Role.ADMIN)
-        if (admins.isEmpty()) {
-            throw UserException("Ни одного администратора не зарегистрировано")
-        }
         if (existingRestaurant == null) {
             val restaurant = restaurantRepository.save(Restaurant(newRestaurant))
-            admins.forEach {
-                notificationService.notify(
-                    it,
-                    "Ресторан с именем ${restaurant.name} был зарегистрирован и ожидает проверки"
-                )
+            if (admins.isNotEmpty()) {
```

Comment:
This check is redundant: if the collection is empty, forEach will not execute, and it won't throw an exception, either